scripts/generate-go-crd-clients/k8s/v1alpha1/register.go

```diff
@@ -30,7 +30,7 @@
 
 // Package v1alpha1 contains API Schema definitions for the k8s v1alpha1 API group
 // +k8s:openapi-gen=false
-// +k8s:deepcopy-gen=
+// +k8s:deepcopy-gen=package,register
```

Comment:
Is this a manual change? I think we cannot override this and the other `register.go` file since they are autogenerated.

Reply:
I think this is the template and the other (pkg/clients/generated/apis/k8s/v1alpha1/register.go) is generated.